src/build/spotbugs-include.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<!--
+~   Licensed under the Apache License, Version 2.0 (the "License");
+~   you may not use this file except in compliance with the License.
+~   You may obtain a copy of the License at
+~
+~   http://www.apache.org/licenses/LICENSE-2.0
+~
+~   Unless required by applicable law or agreed to in writing, software
+~   distributed under the License is distributed on an "AS IS" BASIS,
+~   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~   See the License for the specific language governing permissions and
+~   limitations under the License.
+-->
+<FindBugsFilter xmlns="https://github.com/spotbugs/filter/3.0.0"
+                xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                xsi:schemaLocation="https://github.com/spotbugs/filter/3.0.0 https://raw.githubusercontent.com/spotbugs/spotbugs/3.1.5/spotbugs/etc/findbugsfilter.xsd">
+	<Match>
+		<Package name="~.*"/>
```

Comment:
What does this `~` matching do?
I tried to look up the file format https://spotbugs.github.io/spotbugs-maven-plugin/usage.html but it links to a 404

Reply:
`~` tells it to use regex. I couldn't find out anywhere what exactly will be given to the regex to match against though, but trial and error in my private project showed that `.*` simply matches all your project files (i.e. exactly what we want). But yeah, I don't know what the string is that it's matching either :/ It's a pretty common problem in a lot of options like this one in a lot of these tools. Their docs only mention high-level what will be matched without giving a precise example...

Reply:
I used this for info btw: http://findbugs.sourceforge.net/manual/filter.html

Spotbugs is backward compatible with findbugs, config files included